### PR TITLE
Add mssql parser

### PR DESCRIPTION
**Pull request by phanbaominh**
## Summary
Add mssql parser (using Parsimmon) and importer, parse tests for it.
## Lasting Changes (Technical)

- Add a folder containing mssql parser source code.
- Add mssqlParser.jss in Parse/ (for consistency purpose).

## Checklist

Please check directly on the box once each of these are done

- [X] Documentation (if necessary)
- [X] Tests (integration test/unit test)
- [ ] Integration Tests Passed
- [ ] Code Review

### CODE REVIEW

File: packages/dbml-core/src/parse/mssql/base_utils.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const P = require('parsimmon');
+const wss = require('./whitespaces');
+
+exports.word = function (string) {
+  return P.string(string).skip(wss).desc(`"${string}"`);
+};
+
+function replaceWhitespaceWithRegexp (regexp) {
+  let string = String(regexp);
+  string = string.replace(/[\s]+/g, '[^\\S\\r]+');
```
Comment (phuongduyphan) [inline packages/dbml-core/src/parse/mssql/base_utils.js:10]: Suggest changing this line to this:
```
string = string.replace(/[\s]+/g, '\\s+');
``` 

FYI, `\s` matches white space, including space, tab, line feed, ...
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes

File: packages/dbml-core/src/parse/mssql/base_utils.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  const lastSlash = string.lastIndexOf('/');
+  return new RegExp(string.slice(1, lastSlash), string.slice(lastSlash + 1));
+}
+exports.keyword = function (regexp, multiword = false) {
+  let newRegexp = regexp;
+  const desc = regexp.source;
+  if (multiword) {
```
Comment (phuongduyphan) [inline packages/dbml-core/src/parse/mssql/base_utils.js:17]: Do we really need the checking logic for `multiword` here? How about removing `multiword` param and applying `replaceWhitespaceWithRegexp` for all received regexp?

**Comment by phuongduyphan** [comment]
Suggest moving all the rules from `composite_parsers.js` to `base_parsers.js` em. From my view they're all base parsers.

**Comment by phuongduyphan** [comment]
Suggest moving all the children folders of `statement_types` into `statements` folder as well

**Comment by wingombia** [comment]
> Suggest moving all the rules from `composite_parsers.js` to `base_parsers.js` em. From my view they're all base parsers.

Sould I rename base_parsers to keyword_parsers and composite_parsers to base_parsers instead

**Comment by phuongduyphan** [comment]
> > Suggest moving all the rules from `composite_parsers.js` to `base_parsers.js` em. From my view they're all base parsers.
> 
> Sould I rename base_parsers to keyword_parsers and composite_parsers to base_parsers instead

Yes, i'm ok with that.

**Comment by phuongduyphan** [comment]
@wingombia Also pls remove and ignore `.VSCodeCounter` folder.

**Pull request merged**